sdk/schemaregistry/schema-registry-avro/tests.yml

```diff
@@ -5,3 +5,6 @@ extends:
     parameters:
       PackageName: "@azure/schema-registry-avro"
       ServiceDirectory: schemaregistry
+      MatrixFilters:
+        - TestType=^(?!sample).*
```

Comment:
I think samples should work now. @maorleger to confirm.

Reply:
Ya I expect samples to work now - if they don't then we'd need to look into why

Reply:
Talked offline and @maorleger helped me with updating the imports for the samples to work